vm/mx.vm/mx_vm_benchmark.py

```diff
@@ -645,6 +651,9 @@ def run_stage_agent(self, config, stages):
         hotspot_run_args = []
         hotspot_vm_args += ['-agentlib:native-image-agent=config-output-dir=' + str(config.config_dir), '-XX:-UseJVMCINativeLibrary']
 
+        if config.vm_args is not None:
+            hotspot_vm_args += config.vm_args
```

Comment:
@olpaw could you review the python bits?
Especially to make sure we propagate the add-opens/add-exports at each stage of NI benchmarking where it is needed.

Reply:
Ok. I will take a look.

Reply:
https://github.com/oracle/graal/pull/4000#pullrequestreview-804760921